src/routes/devcards.ts

```diff
@@ -93,6 +93,19 @@ export default async function (fastify: FastifyInstance): Promise<void> {
           return res.status(404).send();
         }
         const type = req.query?.type?.toLowerCase() ?? 'default';
+
+        // for svg, return the same image as png, wrapped in svg tag
+        if (format === 'svg') {
+          const svgString = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><image xlink:href="${req.protocol}://${req.hostname}${req.originalUrl.replace('.svg', '.png')}" /></svg>`;
```

Comment:
Does it include query params? I guess we should include them

Reply:
yes, it does

`originalUrl` unintuitively is everything behind the root path, so `/devcards/v2/.*` for this case.